lxd/node/config.go

```diff
@@ -235,6 +240,16 @@ var ConfigSchema = config.Schema{
 	//  shortdesc: Address to bind the storage object server to (HTTPS)
 	"core.storage_buckets_address": {Validator: validate.Optional(validate.IsListenAddress(true, true, false))},
 
+	// Syslog socket
+
+	// lxdmeta:generate(entity=server, group=core, key=core.syslog_socket)
+	//
+	// ---
+	//  type: bool
+	//  scope: local
+	//  shortdesc: Enable syslog unixgram socket to receive log messages from external processes
```

Comment:
```suggestion
	// lxdmeta:generate(entity=server, group=core, key=core.syslog_socket)
	// Set this option to `true` to enable the syslog unixgram socket to receive log messages from external processes.
	// ---
	//  type: bool
	//  scope: local
	//  shortdesc: Whether to enable logging on the syslog unixgram socket
```

Reply:
I'm not sure about the proposed phrasing as `core.syslog_socket=true` creates the unixgram socket listener, the logs are coming from OVN if configured for it. The config key only controls if LXD will listen/receive those.

Reply:
I think the phrasing is OK. The socket currently only handles OVN logs, but can be extended to handling others as well. I'd keep it generic.

Reply:
Sorry for not being clear, I meant to say that "Whether to enable logging on the syslog unixgram socket" was a bit confusing to me as it does not enable logging but controls the presence of a listener socket.

Reply:
I think it's more clear if one reads the full description as well. Anyway, do you have a suggestion regarding the shortdesc?

Reply:
`shortdesc: Whether to enable the syslog unixgram socket listener`